docs/apm/real-user-monitoring/dashboards.md

```diff
@@ -29,32 +29,38 @@ The data will populate in your organization's **Sumo Logic RUM - default** das
+1. Select your desired dashboard from **Dashboard** dropdown menu in the header:<br/>![explore rum with red box.png](/img/rum/explore-rum-with-red-box.png)<br/> There are three dashboard types on the **Application** and **Service** level and a single one on the **Action type** and **Action** level.
+1. Set your desired filters. **Explore** organizes RUM data on five levels:
+   * **Application**: corresponds to the value of the application tag set in your [RUM script](#step-2-add-script-to-your-page-header). This should correspond to your whole website defined by its business function, such as `the-coffee-bar-app`.
+   * **Service**: corresponds to the name of the service in your [RUM script](#step-2-add-script-to-your-page-header). The value should correspond to the JavaScript code executed in your browser, such as `coffee-bar-frontend`. You can have multiple services for each application.
+   * **deployment.environment**: corresponds to your development environment. To enable this, add the `deployment.environment` tag and desired value (like `us-west-1`, `prod`, `dev`) to your [RUM script](#step-2-add-script-to-your-page-header).
+   * **Action Type**: can be one of:
+     * **document loads**: representing loading of actual documents and their resources into the browser
+     * **XHR actions**: representing any interaction with a page like click or submit that executes AJAX requests in the background to communicate with the backend, or
+     * **route changes**: single-page-app specific way to navigate to a new page/view without having to load a new document.
+   * **Action Name**: automatically generated from URLs. No configuration is required. The specifics of it will depend on action type. Action names can contain asterisks (`*`) to replace automatically-detected dynamic parts of the URL. If you have action names that overlap, the action name with an asterisk contains data for page loads NOT contained in more specific action names. For example, `http://www.site.com/path/page.htm` does not contain actions from `http://www.site.com/path/*`.
+
+Example:<br/><img src={useBaseUrl('img/rum/explore-view-rum.png')} alt="Real User Monitoring" />
 
 
 ## Navigating RUM Dashboards
 
-### RUM Overview Application/Service/Action Type/Action
+### RUM Overview Dashboards
 
-The **RUM Overview Application**, **Service**, **Action Type**, and **Action** dashboards dashboards show the user experience for performance and requests metrics for selected application, service, or action, broken down per top geo-locations, operating systems, and browsers.
+The RUM Overview dashboards show the user experience for performance and requests metrics for your selected application, service, and action. Data is broken down by top geo-locations, operating systems, and browsers. These dashboards include:
+* **RUM Overview - Application**
+* **RUM Overview - Application Service**
+* **RUM Overview - Navigation Action**
+* **RUM Overview - Load Action**
```

Comment:
I dont think it's there

Reply:
@pbrzoska-sumo , can you clarify what action I should take with those four items?

Reply:
correct name is "RUM Overview - DocumentLoad actions"

Reply:
The list is incomplete and ordered in a wrong way. So either fix or remove all together. It is confusing in its current form.